### PR TITLE
Combine stylesheets used for new sign-up flow

### PR DESCRIPTION
**Pull request by TurnerRiley**
Combines the separate stylesheets created while building the separate pages that are part of the new sign-up flow. I removed some of the styles because they were either 1) not doing anything or 2) overlap with styles in other files. I also updated some styles and utilized the `AccountBanner` component to match the Figma better.

### Login type page
Previous:
old_loginType

New:
newLoginType

### Account type page
Previous:
old_accountType

New:
newAccountType

Previous dialog:
old_dialog

New dialog:
new_dialog

### Finish teacher sign-up page
Previous:
old_teacher

New:
newnewTeacher

### Finish student sign-up page
Previous:
old_student

New:
newStudent

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-2215)
Figma: [here](https://www.figma.com/design/88sbry7xnl1wcyrOgbgFtn/Sign-Up-Flow?node-id=2911-1878&t=qFEuJXJfyV0Z1eT3-0)

## Testing story
Local testing to make sure everything is visually the same.

### CODE REVIEW

File: apps/src/signUpFlow/signUpFlowStyles.module.scss [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: 0 20px;
+  }
+
+  .verticalDividerTop,
+  .verticalDividerBottom {
+    width: 1px;
+    background-color: #ccc;
```
Comment (kelbyhawn) [inline apps/src/signUpFlow/signUpFlowStyles.module.scss]: Nit: replace this with the color variable:
```suggestion
    background-color: $light_gray_200;
```

**Review by kelbyhawn — APPROVED** [review]
One tiny nit, but this looks great! Thanks for all of the screenshots 🙌 🎉

File: apps/src/signUpFlow/signUpFlowStyles.module.scss [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+      background: none;
+    }
+
+    & > span {
+      font-weight: 400;
+    }
+  }
+}
+
+.dialogContainer {
```
Comment (kelbyhawn) [inline apps/src/signUpFlow/signUpFlowStyles.module.scss:301]: Oh one more thing I forgot — did you check that the popup dialog looks ok?

File: apps/src/signUpFlow/signUpFlowStyles.module.scss (hunk 1 above)
Comment (TurnerRiley) [reply apps/src/signUpFlow/signUpFlowStyles.module.scss]: Thanks for catching this!

File: apps/src/signUpFlow/signUpFlowStyles.module.scss (hunk 2 above)
Comment (TurnerRiley) [reply apps/src/signUpFlow/signUpFlowStyles.module.scss:301]: Oh yes I did, sorry forgot to attach a screenshot of it. I'll add that now!

**Review by hannahbergam — APPROVED** [review]
Looking forward to this all being merged together! Thank you for doing this!

File: apps/src/signUpFlow/signUpFlowStyles.module.scss [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+.typeHeaderBanner {
+  margin-block: 1.875rem 2.5rem !important;
+
+  h1 {
+    margin-bottom: 0 !important;
+  }
+}
+
+
+// Login Type Selection page
```
Comment (kelbyhawn) [inline apps/src/signUpFlow/signUpFlowStyles.module.scss:55]: Thanks for adding these comments!

**Review by kelbyhawn — APPROVED** [review]
I have one style nit on the Login Method page, but otherwise this looks great! 

I think the spacing here should be equidistant to the space between each label/input pair:
360067181-95a6e524-fc0a-41ed-a017-75bdcfb49ee3

**Comment by TurnerRiley** [comment]
> I have one style nit on the Login Method page, but otherwise this looks great!
> 
> I think the spacing here should be equidistant to the space between each label/input pair: 360067181-95a6e524-fc0a-41ed-a017-75bdcfb49ee3

In the Figma, the button is spaced out evenly between the fields and the header rather than being the same spacing between the fields:
30spacing
@dmcavoy @moshebaricdo Thoughts on this? Happy to go either way!

**Comment by moshebaricdo** [comment]
> > I have one style nit on the Login Method page, but otherwise this looks great!
> > I think the spacing here should be equidistant to the space between each label/input pair: 360067181-95a6e524-fc0a-41ed-a017-75bdcfb49ee3
> 
> In the Figma, the button is spaced out evenly between the fields and the header rather than being the same spacing between the fields: 30spacing @dmcavoy @moshebaricdo Thoughts on this? Happy to go either way!

This is correct! Field wrap should be equal between the heading and the button, but it seems like there might be more spacing between them overall than in the Figma.

One other small nit is that the border around the page content container in the pick your login method screen is using the wrong gray value, should be our 200 level gray (like the two cards on the pick account type screen).

**Comment by TurnerRiley** [comment]
> > > I have one style nit on the Login Method page, but otherwise this looks great!
> > > I think the spacing here should be equidistant to the space between each label/input pair: 360067181-95a6e524-fc0a-41ed-a017-75bdcfb49ee3
> > 
> > 
> > In the Figma, the button is spaced out evenly between the fields and the header rather than being the same spacing between the fields: 30spacing @dmcavoy @moshebaricdo Thoughts on this? Happy to go either way!
> 
> This is correct! Field wrap should be equal between the heading and the button, but it seems like there might be more spacing between them overall than in the Figma.
> 
> One other small nit is that the border around the page content container in the pick your login method screen is using the wrong gray value, should be our 200 level gray (like the two cards on the pick account type screen).

Oh you're right my bad! The button had some snuck in margins that I didn't notice that is adding a tiny bit of extra space. I'll remove that and fix the border color!

**Pull request merged**